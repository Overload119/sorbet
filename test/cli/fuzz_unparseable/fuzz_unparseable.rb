# typed: strict
r *[1]g#typed:true
m do
