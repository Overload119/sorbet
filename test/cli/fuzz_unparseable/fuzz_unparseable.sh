#!/usr/bin/env bash

set -euo pipefail

if main/sorbet --silence-dev-message -a test/cli/fuzz_unparseable/fuzz_unparseable.rb 2>&1; then
  echo "Expected to fail!"
  exit 1
fi
