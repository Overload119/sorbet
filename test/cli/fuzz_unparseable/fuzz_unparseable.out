test/cli/fuzz_unparseable/fuzz_unparseable.rb:2: unexpected token tIDENTIFIER https://srb.help/2001
     2 |r *[1]g#typed:true
              ^

test/cli/fuzz_unparseable/fuzz_unparseable.rb:3: unexpected token "end of file" https://srb.help/2001
     3 |m do
     4 |

test/cli/fuzz_unparseable/fuzz_unparseable.rb:2: Method `g` does not exist on `T.class_of(<root>)` https://srb.help/7003
     2 |r *[1]g#typed:true
              ^

test/cli/fuzz_unparseable/fuzz_unparseable.rb:3: Method `m` does not exist on `T.class_of(<root>)` https://srb.help/7003
     3 |m do
        ^
Errors: 4
